src/main.cpp

```diff
@@ -427,21 +425,32 @@ ast::PassManager CreateAotPM(std::string __attribute__((unused)))
   return pm;
 }
 
-int main(int argc, char* argv[])
+struct Args
 {
-  int err;
   std::string pid_str;
   std::string cmd_str;
   bool listing = false;
   bool safe_mode = true;
   bool usdt_file_activation = false;
   int helper_check_level = 0;
   TestMode test_mode = TestMode::UNSET;
-  std::string script, search, file_name, output_file, output_format, output_elf,
-      aot;
+  std::string script;
+  std::string search;
```

Comment:
If I'm not mistaken, `script` and `search` are mutually exclusive so we could merge them together (I'd use `script` here).

Reply:
They are indeed mutually exclusive, but since they are logically different, I figured it helps readability to keep separate variables.

Again, if the preference is still to merge them, let me know and I'll do it.

Reply:
Ok, let's leave them separate for now, perhaps they'll get merged in future.